.github/workflows/repo-sync.yml

```diff
@@ -0,0 +1,27 @@
+name: Repo Sync
```

Comment:
Should this have the copyright header?

Reply:
As per my knowledge, this action was taken it from this "https://github.com/repo-sync/repo-sync" github action, so we can't add copyright text to this file.